example_configs/ejabberd.md

```diff
@@ -0,0 +1,30 @@
+# Basic LDAP auth for a Ejabberd XMPP server
+
+[Main documentation here.](https://docs.ejabberd.im/admin/configuration/ldap/)
+
+For simple user auth add this to main ejabberd.yml:
+
+```
+host_config:
+  xmpp.example.org:
+    auth_method: [ldap]
+    ldap_servers:
+      - 127.0.0.1
```

Comment:
Let's use "<your ldap host>" here, since it's not always localhost

Reply:
I think having an actual functional value there is easier to understand and modify than something that doesn't explain what exactly to put there, but I will add a comment to explain that this is the ip or hostname of the LLDAP server.